excel-export-service/src/errors.ts

```diff
@@ -0,0 +1,39 @@
+import { NextFunction } from "express";
+import { CustomExpressRequest, CustomExpressResponse } from "./types";
+
+/**
+ * * @extends Error
+ */
+export class ExtendableError extends Error {
+  status: number;
```

Comment:
If status means http status code, shouldn't it be a member of APIError?

Reply:
ExtendableError removed